src/query/src/sql.rs

```diff
@@ -725,6 +726,43 @@ pub fn show_create_view(
     Ok(Output::new_with_record_batches(records))
 }
 
+/// Execute [`ShowViews`] statement and return the [`Output`] if success.
+pub async fn show_views(
+    stmt: ShowViews,
+    query_engine: &QueryEngineRef,
+    catalog_manager: &CatalogManagerRef,
+    query_ctx: QueryContextRef,
+) -> Result<Output> {
+    let schema_name = if let Some(database) = stmt.database {
+        database
+    } else {
+        query_ctx.current_schema()
+    };
+
+    let projects = vec![(tables::TABLE_NAME, VIEWS_COLUMN)];
+    let filters = vec![
+        col(tables::TABLE_SCHEMA).eq(lit(schema_name.clone())),
+        col(tables::TABLE_CATALOG).eq(lit(query_ctx.current_catalog())),
+        col(tables::TABLE_TYPE).eq(lit("VIEW")),
```

Comment:
it would be great if we can replace the hard code here

Reply:
👍 
What about adding a function `name()` to `TableType` like:

https://github.com/GreptimeTeam/greptimedb/blob/main/src/catalog/src/information_schema/tables.rs#L270-L274

Then you can replace `VIEW` with `TableType::View.name()`.

Reply:
Another option is to query from `views`, the PR #4339 was merged.

Reply:
Thanks I used the views virtual table instead.